src/app/contextBroker/contextBroker.cpp

```diff
@@ -1083,7 +1083,6 @@ void orionExit(int code, const std::string& reason)
     destroyAllOntimeIntervalThreads(dbs[ix]);
   }
 
-  mongoDisconnect();
```

Comment:
Reading #929 issues is not a matter of only remove `mongoDisconnect()` but replacing the stament by `setMongoConnectionForUnitTest(NULL)`